### ISSUE TITLE
Add option to add mirror sites

### ISSUE THREAD

**Issue by meanindra** [LagradOst/CloudStream-3#1177]
### Describe your suggested feature

Some sites are blocked by ISP so URL just don't work but some times their mirror URL work fine. If you add option "Change site URL" or "Try mirror site" it would be a useful feature.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Blatzar** [comment]
Closing as this is now added
